lib/users.ts

```diff
@@ -96,17 +105,77 @@ export const getOrCreateUser = async ({
           name: true,
           email: true,
           privileges: true,
+          active: true,
         },
       })
       .catch((e) => prismaErrors(e, null));
   }
 };
 
+/**
+ * Get User by id
+ * @returns User if found
+ */
+export const getUser = async (
+  id: string,
+  ability: UserAbility
```

Comment:
FYI we try to keep 'ability' as the firt argument as a generic pattern for our functions that require ability checks.